xlive/H2Tweaks.cpp

```diff
@@ -515,13 +528,28 @@ int __cdecl validate_and_add_custom_map(BYTE *a1)
 	if (!validate_and_add_custom_map_interal_impl(a1))
 	{
 		TRACE_FUNC("warning \"%s\" has bad checksums or is blacklisted, map may not work correctly", file_name);
+		std::wstring fallback_name;
+		if (strnlen_s(header.name, sizeof(header.name)) > 0)
```

Comment:
open brace on same line

Reply:
or make else open brace on next line

Reply:
one day I will bother to setup msvc correctly